### PR TITLE
Small polish

### PR DESCRIPTION
**Pull request by dev8723**
**Summary**
Hey I came across this repo and it's pretty cool you can store data on babylon with btc timestamped. It's like adding an execution layer (cw) for bitcoin. I tried to make the contract look a bit cleaner and follow cw best practice, but i feel like i went too far and made too many unnecessary changes lol

**What's changed**
1. Renamed some fields and move some things outside `contracts.rs`.
2. Add attributes to execute response, I think it's always nice to emit event in case someone runs indexer.
3. Prefer cw number type instead of rust native number. (There were some issue in the past with serialization issue on native number, so I always prefer to just use cw number type in contract, [post](https://medium.com/cosmwasm/dev-note-4-u128-i128-serialization-in-cosmwasm-90cb76784d44) from cw).
4. Removed the unused `Makefile`.

**Test plan**
1. I'm able to compile the contract successfully, but I didn't deploy it to babylon cause I couldn't find the public testnet info.
2. I looked into writing integration test with cw-multi-test, but turns out I need to mock the babylon binding.

### CODE REVIEW

**Comment by dev8723** [comment]
@maurolacy @vitsalis @SebastianElvis @KonradStaniec 
Feel free to share any feedback, if babylon is looking for cw devs to work as contractor or do some small works (e.g. write more cw contracts like this one), i'm happy to explore those opportunities.

File: src/state.rs
Comment (maurolacy) [inline src/state.rs:6]: 👍🏼

File: src/state.rs
Comment (maurolacy) [inline src/state.rs:7]: This is redundant, as the hash of the data is currently being used as key. But, since it's there, I guess it's not a big deal to return it anyway.

**Review by maurolacy — APPROVED** [review]
Thanks for the contribution.

LGTM.

**Comment by dev8723** [comment]
> Thanks for the contribution.
> 
> LGTM.

Thanks for the review, I don't have write access, feel free to merge for me.

**Comment by maurolacy** [comment]
> > Thanks for the contribution.
> > LGTM.
> 
> Thanks for the review, I don't have write access, feel free to merge for me.

k, will merge after tagging a previous version / release.

**Pull request merged**
